addons/shortcodes.php

```diff
@@ -242,37 +247,27 @@ function shortcode_attendees( $attr ) {
 	 * @return string
 	 */
 	protected function get_avatar_placeholder( $id_or_email ) {
-		$url = get_avatar_url( $id_or_email );
-
-		// Remove query args
-		$url = substr_replace( $url, '', strpos( $url, '?' ) );
-
-		// Isolate the hash
-		$gravatar_hash = substr( $url, strpos( $url, '/avatar/' ) + 8 );
+		// @todo Allow customization of avatar and placeholder size
+		$size = 96;
 
 		return sprintf(
-			'<div class="avatar avatar-placeholder" data-tix-avatar-hash="%s" data-appear-top-offset="500"></div>',
-			$gravatar_hash
+			'<div class="avatar avatar-placeholder" data-url="%s" data-url2x="%s" data-size="%s" data-alt="%s" data-appear-top-offset="500"></div>',
+			get_avatar_url( $id_or_email ),
+			get_avatar_url( $id_or_email, array( 'size' => $size * 2  ) ),
+			$size,
+			''
```

Comment:
I think something like `data-alt="Avatar for Jane Doe"` would improve accessibility.